SSA/Projects/InstCombine/Base.lean

```diff
@@ -353,6 +353,7 @@ instance : Lean.ToFormat LLVM.Ty := by unfold LLVM; infer_instance
 
 instance : ToString (MTy φ) where
   toString t := repr t |>.pretty
+
```

Comment:
```suggestion
```

Reply:
There is still one unrelated change.

Reply:
For such whitespace errors, just open a separate PR and merge it "as obvious" without waiting for a review.